docs/config/dfdaemon_config_template.yml

```diff
@@ -1,42 +1,51 @@
 # This file is the template of dfdaemon configuration file.
 # You can configure your dfdaemon by change the parameter according your requirement.
 
-# RegistryMirror configures the mirror of the official docker registry
```

Comment:
Can we add yamllint in CI to ensure that all the yaml files are correctly formatted?

Reply:
Is it https://github.com/adrienverge/yamllint?

Reply:
Right, that's the one!